versioned_docs/version-0.4.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx

```diff
@@ -13,19 +13,19 @@ see [Prerequisites](/versioned_docs/version-0.4.0/getting-started-with-cloud-clu
 see [Preparing the clusters for registration](/versioned_docs/version-0.4.0/getting-started-with-cloud-clusters/prerequisites/preparing-the-cloud-clusters-for-registration.mdx).
 - You have installed Istio on the worker clusters to configure the external gateways. For more information, 
 see [Installing Istio](/versioned_docs/version-0.4.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller.mdx#install-istio).
-- You have cluster administrator privileges to install the KubeSlice Controller on controller cluster and Slice Operator on worker clusters.
+- You have cluster administrator privileges to install the KubeSlice Controller on controller cluster and the Slice Operator on worker clusters.
 
 ## Install KubeSlice
 
-In this demonstration install KubeSlice on cloud clusters, using the custom topology configuration YAML.
+In this demonstration, install KubeSlice on cloud clusters using the custom topology configuration YAML.
```

Comment:
In this demonstration, let us install .. 

In this demo, install (would make the user action imperative). Since we are guiding, use `let us`...

Reply:
done

Reply:
done

Reply:
done